py/htm/examples/hotgym.py

```diff
@@ -63,8 +63,7 @@ def main(parameters=default_parameters, argv=None, verbose=True):
     potentialPct               = spParams["potentialPct"],
     potentialRadius            = encodingWidth,
```

Comment:
btw, shouldn't this be encodingWidth/2 ?

Reply:
IIRC the SP will reduce this parameter to an acceptable value, so this is really asking the SP "make all-to-all connections"

Reply:
ok, I thought there might be more inputs, and this would be stretching over that one..